README.md

```diff
@@ -111,3 +111,91 @@ Quering schema field types (the order matches `get_names`):
 ```lua
 avro_schema.get_types(schema)
 ```
+
+## References
+
+Named types are ones that have mandatory `name` field in its definition:
+record, fixed, enum.
+
+Named types can be referenced after the first definition (forward declarations
+are forbidden). Of course, primitive types can be referenced in any place of a
+schema.
+
+Note 1: A reference is a usage of a type (not a value), so the effect is like
```

Comment:
Not sure about that. I think that it is not true.

Reply:
Need to be clarified, sure.